templates/error/error.md

```diff
@@ -0,0 +1,13 @@
+# {{{ name }}}
+
+{{#if codeMessage}}
+{{{ codeMessage }}}
+{{else}}
+{{{ message }}}
+{{/if}}
+
+{{#if stack}}
```

Comment:
Let's wrap that in a `<pre>`.

Reply:
The markdown to html conversion library takes care of that due to the code tags.